scrapyd_k8s/api.py

```diff
@@ -25,7 +25,17 @@ def healthz():
 
 @app.get("/daemonstatus.json")
 def api_daemonstatus():
-    return { "status": "ok", "spiders": 0 }
+    jobs = launcher.listjobs()
+    total_pending = len([j for j in jobs if j['state'] == 'pending'])
+    total_running = len([j for j in jobs if j['state'] == 'running'])
+    total_finished = len([j for j in jobs if j['state'] == 'finished'])
```

Comment:
I think these can also move to the return statement, right?

Reply:
It's just a matter of codestyle, terseness vs. smaller bits per line. Thanks for fixing!